.github/benchmark-files/rector.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
This file is used to run Rector to measure the performance of PHP for CLI tools. It just runs a small set of rules against a single file in order to make the test reasonably fast for our benchmark